modules/ROOT/pages/_partials/showcase-apps/service-setup.adoc

```diff
@@ -36,48 +37,170 @@ Showcase apps demonstrate many features of {product-name}. To see these features
+|Sync Application metrics endpoint| /metrics
+|Sync App GraphQL Endpoint| /graphql
+|Database Server Hostname| Hostname of the PostgreSQL server, usually *postgresql*
+|Database Server Port| Port number of the PostgreSQL server, usually *5432*
+|Database Server Username| *Username of the PostgreSQL database connection*
+|Database Server Password| *Password of the PostgreSQL database connection*
+|Database name| *PostgreSQL database name*
 |====
-.. xref:data-sync.adoc#sync-server-binding-keycloak[Protect the {sync-service} app using {keycloak-service}]
++
+. Provision the Mobile Developer Console
+.. Click `Browse Catalog` and select `Mobile` -> `Mobile Developer Console`.
+.. Complete the wizard to start the deployment of the *Mobile Developer Console*.
+
+[[creating-the-data-sync-mobile-app-binding]]
+=== Creating the Data Sync Mobile App binding
+
+[discrete]
+.Prerequisites
+
+xref:setting-up-mobile-services-to-demonstrate-showcase-apps[Setting up Mobile Services to Demonstrate Showcase Apps]
+
+[discrete]
+.Procedure
+
+. Log in to the Mobile Developer Console by clicking on the Mobile Developer Console link within Openshift.
+. Click *Create Mobile App*, and complete the wizard to create a Mobile App.
+. Click on the newly created *Mobile App* and click *Mobile Services*.
+. The Data Sync provisioned service should be available under the _Unbound Services_ heading. Click *Bind to App* and follow the wizard.
+. When the binding completes, click on *configuration*. The configuration for the Data Sync Mobile App is displayed, for example:
++
+[source,javascript]
+----
+{
+  "version": 1,
+  "namespace": "test12",
+  "clientId": "test16",
+  "services": [
+    {
+      "id": "8565c444-6bed-33e9-8616-0af08791569c",
+      "name": "sync-app-syncshowcase-test16",
+      "type": "sync-app",
+      "url": "https://sync-app-syncshowcase.testbed.com/graphql",
+      "config": {
+        "websocketUrl": "wss://sync-app-syncshowcasetestbed.com/graphql"
+      }
+    }
+  ]
+}
+
+module.exports = config;
+----
+
+[[configuring-the-showcase-app-to-point-at-the-provisioned-data-sync-service]]
+=== Configuring the Showcase App to point at the provisioned Data Sync service
+
+[discrete]
+.Prerequisites
+
+xref:creating-the-data-sync-mobile-app-binding[Creating the Data Sync Mobile App binding]
+
+[discrete]
+.Procedure
+
+1. Clone the showcase app
++
+include::../cloning-showcase-app.adoc[]
++
+2. Incorporate the Data Sync Mobile App code into the *mobile-services.json* file located within _src_:
++
+[source,javascript]
+----
+let config = {
+  "version": 1,
+  "namespace": "test12",
+  "clientId": "test16",
+  "services": [
+    {
+    "id": "8565c444-6bed-33e9-8616-0af08791569c",
+    "name": "sync-app-syncshowcase-test16",
+    "type": "sync-app",
+    "url": "https://sync-app-syncshowcase.testbed.com/graphql",
+    "config": {
+      "websocketUrl": "wss://sync-app-syncshowcasetestbed.com/graphql"
+      }
+}
+  ]
+};
+
+module.exports = config;
+----
```

Comment:
Sorry, the change of the file type also affects the content of a file a little bit. See the [mobile-services.json example](https://github.com/aerogear/ionic-showcase/blob/master/src/mobile-services.json)

Reply:
Hi @psturc - does this address your comment: https://github.com/aerogear/mobile-docs/pull/682/commits/17d900f90da560b5a188728ed4a427162d898926 ?

Reply:
I meant to rewrite [this JS example](https://github.com/aerogear/mobile-docs/pull/682/files#diff-6548f1cccbdf3e920c68badb77e6ae10R137) to JSON, which means to remove the JS-specific stuff from the example, i.e. removing the variable `let config`, `module.exports = config;` etc.

Reply:
Ah, ok. @wtrocki - can you help me out with this - the javascript code below requires conversion to json (if you do this in a PR, it would be awesome!!! ): 

[source,javascript]
----
{
  "version": 1,
  "namespace": "test12",
  "clientId": "test16",
  "services": [
    {
      "id": "8565c444-6bed-33e9-8616-0af08791569c",
      "name": "sync-app-syncshowcase-test16",
      "type": "sync-app",
      "url": "https://sync-app-syncshowcase.testbed.com/graphql",
      "config": {
        "websocketUrl": "wss://sync-app-syncshowcasetestbed.com/graphql"
      }
    }
  ]
}
 module.exports = config;
----

Reply:
@jstaffor oh, sorry, it is actually almost identical: https://gist.github.com/psturc/e30271f9c72aa0980ccdfa706ac94732

Reply:
Thanks @psturc -  If you are happy with https://github.com/aerogear/mobile-docs/pull/682/commits/7e440f3d15d1e47ba0c06d531500d76043d7584f , I think we are ready to merge :+1: